api/models/topics-models.js

```diff
@@ -6,4 +6,22 @@ exports.selectTopics = () => {
     return db.query(queryString).then((result) => {
         return result.rows
     })
-}
\ No newline at end of file
+}
+
+exports.selectArticlesById = (article_id) => {
+    let queryString = `
+    SELECT * FROM articles
+    `
+    if (!Number(article_id)){
+        return Promise.reject({ status: 400, msg: "Bad Request" });
+    }
+    queryString += `WHERE article_id = ${article_id} ;`
```

Comment:
This should absolutely not be in template literals, this leaves you open for SQL injection.